packages/palette/src/elements/Modal/Modal.tsx

```diff
@@ -230,8 +230,16 @@ const ModalElement = styled(AnimatedView)<TransitionElementProps>`
 const ModalScrollContent = styled(Box)<ModalScrollContentProps>`
   height: 100%;
   width: 100%;
-  overflow: scroll;
-  padding-bottom: ${props => (!!props.FixedButton ? "112px" : space(2) + "px")};
+  overflow: auto;
+  padding-bottom: ${space(2)}px;
+  max-height: ${props =>
+    !!props.FixedButton
+      ? "calc(100vh - (80px + " + space(9) + "))"
```

Comment:
```suggestion
      ? "calc(100vh - (80px + " + space(9) + "px" +  "))"
```

Reply:
I think it might be a little easier to grok if it was

```
`calc(100vh - (80px ${space(9)}px))`
```

You can nest <code>`</code> when it's inside an interpolation `${ ... }`

Reply:
This fixes the issue that @dleve123 found/x-posted above (which only affects the modal when using the `FixedButton`)

Reply:
Thanks for the catch, just updated.